docs/pages/versions/unversioned/sdk/screen-capture.md

```diff
@@ -88,6 +88,14 @@ import {
 
 ## Methods
 
+### `isAvailableAsync()`
+
+Returns whether the Screen Capture API is enabled on the current device.
+
+#### Returns
+
+A `Promise` resolving to either `true` or `false`.
```

Comment:
```suggestion
Async `boolean`, indicating whether the Screen Capture API is available on the current device. Currently this resolves `true` on iOS and Android only.
```

Reply:
Thanks, I'll try to keep the same format you mention here in other PRs if I open them!